doc/source/release-notes/1.2.2.rst

```diff
@@ -0,0 +1,60 @@
+.. _1.2.2:
+
+=======================================
+Percona Monitoring and Management 1.2.2
+=======================================
+
+:Date: August 23, 2017
+
+For install and upgrade instructions, see :ref:`deploy-pmm`.
+
+This release contains bug fixes related to performance and introduces
+various improvements. It also contains an updated version of Grafana.
+
+
+Changes in PMM Server
+=====================
+
+The following changes were introduced in *PMM Server* 1.2.2:
+
+Bug fixes
+---------
+
+* :pmmbug:`927`: The error "Cannot read property 'hasOwnProperty' of
+  undefined" was displayed on the QAN page for mongodb.
+
+  After enabling monitoring and generating data for mongodb, the PMM client
+  showed the following error message on the QAN page: "Cannot read property
+  'hasOwnProperty' of undefined". This bug is now fixed.
+
+* :pmmbug:`949`: Percona Server was not detected properly, the log_slow_* variables were not properly detected.
+
+* :pmmbug:`1081`: QAN2 showed very high load for MariaDB mdb101 host on pmmdemo
```

Comment:
It's more about Performance Schema monitoring problem, not MariaDB only. 
The  more explanations in https://github.com/percona/qan-agent/pull/53